src/main/kotlin/traffic_simulation/Vehicle.kt

```diff
@@ -1,31 +1,10 @@
 package traffic_simulation
 
-class Vehicle(val id : Int, val wannaDrive : Boolean, var delayed : Boolean = false, var hasDriven : Boolean = false){
-    val allVehicles : MutableList<Vehicle> = mutableListOf()
+class Vehicle(val id : Int, val wannaDrive : Boolean){
+    var delayed : Boolean = false //no vehicle is delayed by default
```

Comment:
Sowohl die Auflistung in Klammern hinter dem Klassennamen als Variablen, die außerhalb einer Methode deklariert werden, sind Attribute. Der Unterschied ist nur, dass man die einen mit dem Konstruktor setzen kann und die anderen nicht.

D.h. `delayed` ist hier ein Attribut.